File: exercises/02_exercise_conditional_statements/08_lunch_break.py
import math

name_series = str(input())
duration_episode = int(input())
duration_break = int(input())
duration_lunch = duration_break / 8
duration_rest = duration_break / 4
left_time = duration_break - duration_lunch - duration_rest
needed_time = abs(left_time - duration_episode)
if left_time >= duration_episode:
    print(f"You have enough time to watch {name_series} and left with {math.ceil(needed_time)} \
minutes free time.")
else:
    print(f"You don't have enough time to watch {name_series}, you need {math.ceil(needed_time)} more \
minutes.")

File: 05_JavaScriptBasics/02_Exercises/02_ConditionalStatements/08_lunchBreak.js
function lunchBreak(input){
    let nameSeries = input[0]
    let durationEpisode = Number(input[1])
    let durationBreak = Number(input[2])

    let lunchTime = durationBreak / 8
    let restTime = durationBreak / 4
    let freeTime = durationBreak - lunchTime - restTime
    let difference = Math.abs(freeTime -  durationEpisode)

    if (freeTime >= durationEpisode){
        console.log(`You have enough time to watch ${nameSeries} and left with ${Math.ceil(difference)} minutes free time.`)
    } else {
        console.log(`You don't have enough time to watch ${nameSeries}, you need ${Math.ceil(difference)} more minutes.`)
    }
}
